salt/netapi/rest_cherrypy/app.py

```diff
@@ -1888,6 +1888,8 @@ def POST(self, **kwargs):
 
             if token["eauth"] == "django" and "^model" in eauth:
                 perms = token["auth_list"]
+            elif token["eauth"] == "rest":
```

Comment:
I'm seeing this error when I test this fix:


```
Traceback (most recent call last):
  File "/home/ch3ll/git/salt/salt/netapi/rest_cherrypy/app.py", line 1903, in POST
    perms = token["auth_list"]
KeyError: 'auth_list'
```

If found out this occurs if `keep_acl_in_token` is False, which it is by default. When I set it to True, then this fix worked for me.

Reply:
Yes, `keep_acl_in_token` would be `True` for rest_cherrypy

Reply:
@Ch3LL Can this be fixed with simply adding a check for it, as in:
```python
elif token["eauth"] == "rest" and "auth_list" in token:
```
or would it be preferable to look it up against the REST API in that case?